testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -399,5 +400,7 @@
 ALIGN_BOUND = resource_filename(__name__, 'data/analysis/align_bound.pdb.gz')
 ALIGN_UNBOUND = resource_filename(__name__, 'data/analysis/align_unbound.pdb.gz')
 
+GSD = resource_filename(__name__, 'data/example.gsd')
```

Comment:
the test file is 15.6MB, can this be trimmed to a smaller system/fewer frames

Reply:
Sure, fixed